src/frontend/src/flows/dappsExplorer/teaser.ts

```diff
@@ -0,0 +1,85 @@
+import { html, TemplateResult } from "lit-html";
+import { dapps } from "./dapps";
+
+/** A teaser for the dapps explorer, acting as a button */
+export const dappsTeaser = ({
+  click,
+}: {
+  click: () => void;
+}): TemplateResult => {
+  return html`<article class="c-card c-card--narrow">
+    <h3 class="t-title t-title--discrete" role="presentation">
+      Dapps explorer
```

Comment:
i18n? In another PR?

Reply:
Yes, will do it as part of i18n-izing the management page. Good point though.